src/plugins/lua/known_senders.lua

```diff
@@ -197,6 +210,57 @@ local function known_senders_callback(task)
   end
 end
 
+local function check_known_incoming_mail_callback(task)
+  local sender = task:get_reply_sender()
+  if not sender then
+    lua_util.debugm(N, task, 'Could not get sender')
+    return nil
+  end
+
+  -- making sender key
+  lua_util.debugm(N, task, 'Sender: %s', sender)
+
+  local sender_string = lua_util.maybe_obfuscate_string(tostring(sender), settings, settings.sender_prefix)
+  local sender_key = make_key_replies(sender_string:lower(), 8)
+
+  lua_util.debugm(N, task, 'Sender key: %s', sender_key)
+
+  local list_of_senders = {}
```

Comment:
What is the purpose of this variable, counting that we just assign it to `data` in the callback afterwards?